main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
+// Displays the story based on the users input
+    print ("------------------------------------------")
```

Comment:
While `print ()` does work, `print()` is preferred. Python stylization recommends you eliminate whitespace:
```suggestion
    print("------------------------------------------")
   ```
[Read more here](https://peps.python.org/pep-0008/#pet-peeves) about the PEP 8 style guide's recommendations. 
> Immediately before the open parenthesis that starts the argument list of a function call:
> ```
> # Correct:
> spam(1)
> ```
> ```
> # Wrong:
> spam (1)
> ```
This affects lines 14-19 as well.